pkg/nodeidentity/aws/identify.go

```diff
@@ -35,6 +35,7 @@ import (
 // CloudTagInstanceGroupName is a cloud tag that defines the instance group name
 // This is used by the aws nodeidentifier to securely identify the node instancegroup
 const CloudTagInstanceGroupName = "kops.k8s.io/instancegroup"
+const CloudTagLabelPrefix = "kops.k8s.io/label/"
```

Comment:
We might want to use `"k8s.io/cluster-autoscaler/node-template/label/"` instead, per https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#how-can-i-scale-a-node-group-to-0